apps/src/applab/designElements/ImagePickerPropertyRow.jsx

```diff
@@ -9,12 +16,31 @@ var PropertyRow = React.createClass({
 
   getInitialState: function () {
     return {
-      value: this.props.initialValue
+      value: this.props.initialValue,
+      lastEdit: new Date(0)
```

Comment:
yeah, Date(0) creates a date at 0 UTC ms. Date() creates a date object at time now. I could probably instead use Date.now() instead of new Date().